packages/renderer/src/App.svelte

```diff
@@ -70,8 +70,11 @@ window.events?.receive('display-help', () => {
 </script>
 
 <Route path="/*" breadcrumb="Home" let:meta>
-  <main class="min-h-screen flex flex-col h-screen bg-zinc-900">
-    <header id="navbar" class="text-gray-700 bg-zinc-900 body-font" style="-webkit-app-region: drag;">
+  <main class="min-h-screen flex flex-col h-screen bg-charcoal-800">
+    <header
+      id="navbar"
+      class="text-gray-700 bg-charcoal-900 body-font shadow-titlebar z-999;"
```

Comment:
do we need the z-index for the shadows ?

Reply:
@mairin ?

Reply:
Yeh, it's definitely needed. Without raising the z-index, you can't see the highlight line along the screen at the bottom of the titlebar. Now, whether or not it needs to be 999 specifically is debatable 😃